### ISSUE TITLE
Crash with undo in level 2.212

### ISSUE THREAD

**Issue by Wuzzy2** [Der1248/Labyrinthus#4]
If you play in level 2.212 and press the Undo key, the game crashes.

How to crash: Move one block to the right, then undo. The timer block mysteriously disappears. Then move again. The game crashes. Note this happens without activating the timer block.

Labyrinthus 2.15.1, Minetest 5.9.1.

Crash log:

```
2024-10-05 19:30:52: ERROR[Main]: ServerError: AsyncErr: Lua: Runtime error from mod 'labyrinthus' in callback environment_Step(): ...zy/.minetest/games/labyrinthus/mods/labyrinthus/init.lua:989: attempt to concatenate a nil value
2024-10-05 19:30:52: ERROR[Main]: stack traceback:
2024-10-05 19:30:52: ERROR[Main]: 	...zy/.minetest/games/labyrinthus/mods/labyrinthus/init.lua:989: in function 'move_data'
2024-10-05 19:30:52: ERROR[Main]: 	...zy/.minetest/games/labyrinthus/mods/labyrinthus/init.lua:353: in function 'globalstep'
2024-10-05 19:30:52: ERROR[Main]: 	...zy/.minetest/games/labyrinthus/mods/labyrinthus/init.lua:306: in function <...zy/.minetest/games/labyrinthus/mods/labyrinthus/init.lua:303>
2024-10-05 19:30:52: ERROR[Main]: 	...uzzy/src/minetest/git/bin/../builtin/common/register.lua:26: in function <...uzzy/src/minetest/git/bin/../builtin/common/register.lua:12>
```

BTW: I suggest to disable the undo functionality once the timer has been activated.

**Comment by Der1248** [comment]
I will try to fix that for the upcoming version 2.16.0

**Comment by Der1248** [comment]
Should be fixed with Labyrinthus 2.16.0.
